Description: Fix code to compile with wxwidgets 3.0
 Drops support for wx < 2.7, which is just not relevant now.  And in fact
 configure.ac checks for at least 2.8.0.
Author: Olly Betts <olly@survex.com>
Last-Update: 2014-03-07

--- src/chmframe.cpp
+++ src/chmframe.cpp
@@ -265,11 +265,7 @@ void CHMFrame::OnChangeFonts(wxCommandEv
 		enu.EnumerateFacenames();
 		_normalFonts = new wxArrayString;
 
-#if wxMAJOR_VERSION == 2 && wxMINOR_VERSION >= 7
 		*_normalFonts = enu.GetFacenames();
-#else
-		*_normalFonts = *enu.GetFacenames();
-#endif
 		_normalFonts->Sort();
 	}
 
@@ -278,11 +274,7 @@ void CHMFrame::OnChangeFonts(wxCommandEv
 		enu.EnumerateFacenames(wxFONTENCODING_SYSTEM, TRUE);
 		_fixedFonts = new wxArrayString;
 
-#if wxMAJOR_VERSION == 2 && wxMINOR_VERSION >= 7
 		*_fixedFonts = enu.GetFacenames();
-#else
-		*_fixedFonts = *enu.GetFacenames();
-#endif
 		_fixedFonts->Sort();
 	}
 
